Directory.Build.props

```diff
@@ -5,5 +5,6 @@
     <IsPackable>false</IsPackable>
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <LangVersion>10.0</LangVersion>
+	<NoWarn>NU5100,NU5104,NU5123,NU5128</NoWarn>
```

Comment:
nit: indentation
```suggestion
	  <NoWarn>NU5100,NU5104,NU5123,NU5128</NoWarn>
```

Reply:
In case someone wants to check what are those:

- https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5100
- https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5104
- https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5123
- https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128

Reply:
Can we for now, remove NU5104 and mark our nuget as prerelease also?
If needed it can be reintroduced later.

Reply:
Removed NU5104 in new implementation.